client/onboarding/utils.ts

```diff
@@ -42,6 +42,18 @@ export const getBusinessTypes = (): Country[] => {
 	);
 };
 
+export const getMccFromIndustry = (): string | undefined => {
+	const industry = wcSettings.admin.onboarding.profile.industry?.[ 0 ];
+	if ( ! industry ) {
+		return undefined;
```

Comment:
Nitpick: IMO, returning `null` will be more explicit. Eg.

```javascript
const initialData = {
	business_name: wcSettings?.siteTitle,
	URL: blaBla
};

console.log( initialData.mcc ); // undefined (whoops! we forgot to define 'mcc', good catch!)
```

```javascript
const initialData = {
	business_name: wcSettings?.siteTitle,
	mcc: getMccFromIndustry(), // returns undefined
	URL: blaBla
};

console.log( initialData.mcc ); // undefined (wait, but I did define 'mcc' above)
```

Of course, as with many things in JS, there's no right or wrong here ;)

Reply:
This is a good point.

I wasn't sure if the [logic](https://github.com/Automattic/woocommerce-payments/blob/1375e040db9711770f741d508794a93a47927d8e/client/onboarding/context.tsx#L12-L12) this eventually gets passed to would safely treat null as "no value" (not prepoulate the form value) 

@ismaeldcom, you wouldn't happen to remember if it's safe to pass a null value into the `initialData` array, and it will just be ignored?

Reply:
Yep, it should be safe to use `null`, although you'll need to update all the type definitions to support it, like `type OnboardingFields`.

I personally prefer `undefined` over `null`, it's easy to mix them with JS, but they are different types. I like the optional syntax of TS for `undefined` and the fact that it gets removed.

When we need to define an interface I prefer `mcc?: string` over `mcc?: string | null` or `mcc: string | null | undefined`.

Anyway, it's a matter of taste 🙂

Reply:
Thanks Ismael, that's very helpful! 

@anu-rock - I'm inclined to keep the logic as is, since I like the simplicity that using `undefined` here gives us - on the type definition for `OnboardingFields` we specify `mcc?: string`, for example - which is consistent with the other fields. I added a docblock to make this more clear in bdd1a35c8.